spec/imsc-hrm.html

```diff
@@ -247,9 +247,164 @@ <h2>Terms and Definitions</h2>
     Instance</a> yields results that might not reflect the complexity of the <a>Document Instance</a>.</p>
   </section>
 
+  <section class="informative">
+    <h2>Introduction</h2>
+  
+    <section>
+      <h3>Objective</h3>
+  
+      <p>The objective of the HRM is to allow subtitle and caption authors
+        and providers to verify that the content they provide does not exceed
+        defined complexity levels, so that playback systems can render the content
+        synchronized with the author-specified display times.</p>
+  
+      <p>Playback systems include desktop computers, mobile devices and
+        home theatre devices.</p>
+
+      <p class="note">The HRM is not a new concept: it has been included in all
+      versions and editions of [[IMSC]] and has remained substantially
+      unchanged. It is refactored herein to simplify document maintenance.</p>
+    </section>
+  
+    <section>
+      <h3>Why limit the complexity of <a>IMSC Document Instances</a>?</h3>
+  
+      <p><a>IMSC Document Instances</a> are typically authored by a first party
+      and rendered by a second party. Unless both parties agree on the maximum
+      complexity of a <a>IMSC Document Instance</a>, it is likely that:</p>
+  
+      <ul>
+        <li>an <a>IMSC Document Instance</a> authored by a first party will
+        exceed the capabilities of the <a>presentation processor</a> of a second
+        party, resulting in an incomplete presentation where some subtitles or
+        captions might be missing or might be presented too late;</li>
+
+        <li>a first party authors only extremely simple <a>IMSC Document
+        Instance</a> in an attempt to ensure a complete presentation across all
```

Comment:
Grammar seems a bit odd - suggest making Document Instance plural, or adding the word "an" before "extremely".

```suggestion
       Instances</a> in an attempt to ensure a complete presentation across all
```